frontend/src/app/core/services/auth.service.ts

```diff
@@ -81,6 +81,12 @@ export class AuthService {
     }
 
     loadUser() {
+        const userFromStorage = localStorage.getItem('user');
+
+        if (userFromStorage) {
+            return of(JSON.parse(userFromStorage));
+        }
+
```

Comment:
No, no, no, please no
This function is used to update user, if you will make so, it won't be possible anymore

Reply:
fixed